### ISSUE TITLE
Message showing MediaInfo version 0.7.64 or newer required, but I have v0.7.82

### ISSUE THREAD

**Issue by agalasso** [xbgmsharp/piwigo-videojs#107]
Hi,

I've got an Ubuntu Xenial (16.04.2 LTS) server with:

> $ mediainfo --version
> MediaInfo Command line, 
> MediaInfoLib - v0.7.82

and I'm getting this message:

> Please use at least MediaInfo version 0.7.64 or higher, not 0.1

when I try to synchronize videos and when I go to the VideoJS tab of the properties page (and I don't get the metadata like date_creation.)

Any idea how I can fix this?

Thanks!

**Comment by agalasso** [comment]
some additional info:

```
$ mediainfo  --Full --Output=XML --Language=raw  20170521_114317.mp4
<?xml version="1.0" encoding="UTF-8"?>
<Mediainfo
    version="0.1"
    ref="20170521_114317.mp4">
<File>
<track type="General">
<Count>323</Count>
...
```

**Comment by agalasso** [comment]
... so apparently a mediainfo bug?

I upgraded to v0.7.96 and all is well.

**Comment by xbgmsharp** [comment]
Improve check to ensure valid working version.
Fix with v2.9.a release.
